package.json

```diff
@@ -34,11 +34,12 @@
     "react-icons": "^4.3.1",
     "react-popper": "^2.2.5",
     "react-router-dom": "^5.3.0",
-    "react-scripts": "4.0.3",
+    "react-scripts": "4.0.1",
```

Comment:
Why did you need to downgrade the lib?

Reply:
I don't remember now, but I think I was facing a bug and forgot to remove it 😅.  I'll remove.